profiles/drupalru/modules/dru_tnx/includes/dru_tnx.crud.inc

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @file
+ *   Data Base functionality
+ */
+
+/**
+ * Implementation get_user_tnx_statistic().
+ *
+ * Return user statistic object
+ */
+function get_user_tnx_statistic($uid) {
```

Comment:
Name of the function isn't available for the module. You need to use names which started from module name: "dru_thx".